mappings/net/minecraft/entity/ai/goal/GoalSelector.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_2641 net/minecraft/GoalSelector
```

Comment:
Whenever you name a class and move it to a folder, remember to add the path within the mapping file too; this can be done in the same text input that's used to rename the class within Enigma